restcomm/restcomm.telephony.api/src/main/java/org/restcomm/connect/telephony/api/util/CallControlHelper.java

```diff
@@ -47,6 +47,10 @@
 public class CallControlHelper {
 
     static boolean permitted(final String authorization, final String method, DaoManager daoManager) {
+        return permitted(authorization, method, daoManager, "MD5");
```

Comment:
@abdulazizali77 why do we hardcode to MD5 instead of using client-algorithm from the settings ?